src/cmd2.cc

```diff
@@ -717,41 +657,6 @@ static void chest_death(int y, int x, s16b o_idx)
 
 
 /*
- * Chests have traps too.
```

Comment:
If I recall the code section right, it might just be naming. I'll see if I can reproduce it.